components/schemes/Color/src/MDCSemanticColorScheme.h

```diff
@@ -74,6 +74,18 @@
  @c backgroundColor.
  */
 @property(nonnull, readonly, copy, nonatomic) UIColor *onBackgroundColor;
+
+/**
+ A color used on the overlay when a surface is elevated.
+ */
+@property(nonnull, readonly, copy, nonatomic) UIColor *elevationOverlayColor;
+
+/**
+ A flag that indicates whether the overlay color should be applied on an elevated surface in Dark
```

Comment:
Note: There is no elevation overlay applied outside of Dark Mode.

Reply:
Addressed in 0ee543f80c066811a44a4e8b54b788e22f2c0c4a